purchase_stock_picking_invoice_link/tests/test_purchase_stock_picking_invoice_link.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 2019 Vicent Cubells <pedro.baeza@tecnativa.com>
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from odoo.tests import Form, common
+
+
+class TestPurchaseSTockPickingInvoiceLink(common.SavepointCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.supplier = cls.env["res.partner"].create({"name": "Supplier for Test"})
+        cls.product = cls.env["product.product"].create({"name": "Product for Test"})
+        po_form = Form(cls.env["purchase.order"])
+        po_form.partner_id = cls.supplier
+        with po_form.order_line.new() as po_line_form:
+            po_line_form.product_id = cls.product
+            po_line_form.price_unit = 15.0
+        cls.po = po_form.save()
+
+    def test_puchase_stock_picking_invoice_link(self):
+        # Purchase order confirm
+        self.po.button_confirm()
+        # Validate shipment
+        picking = self.po.picking_ids[0]
```

Comment:
```suggestion
        picking = fields.first(self.po.picking_ids)
```

Reply:
I think it's no necessary to import fields and use  fields.first in this case because I'm sure there is one and only one record in the recordset self.po.picking_ids. What do you think?

Reply:
It is just a suggestion, it is not necessary to change it and it does not block anything.